File: artist-service/src/main/java/com/gallery/artistservice/domain/ArtistID.java
package com.gallery.artistservice.domain;

public class ArtistID {
    private int id;

    public ArtistID() {
        this.id = 0;
    }

    public ArtistID(int id) {
        this.id = id;
    }

    public ArtistID(ArtistID other) {
        this.id = other.id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }
}

File: app/ArtistService/Domain/ArtistID.cs
namespace ArtistService.Domain
{
    public class ArtistID
    {
        private int _artistId;

        public ArtistID()
        {
            this._artistId = 0;
        }

        public ArtistID(int artistId)
        {
            this._artistId = artistId;
        }

        public ArtistID(ArtistID artistID)
        {
            this._artistId = artistID._artistId;
        }

        public int ArtistId
        {
            get { return this._artistId; }
            set { this._artistId = value; }
        }
    }
}